docs/types/value-types.rst

```diff
@@ -161,19 +161,33 @@ Fixed Point Numbers
     cannot be assigned to or from.
 
 ``fixed`` / ``ufixed``: Signed and unsigned fixed point number of various sizes. Keywords ``ufixedMxN`` and ``fixedMxN``, where ``M`` represents the number of bits taken by
-the type and ``N`` represents how many decimal points are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
+the type and ``N`` represents how many fractional decimal digits are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
 ``ufixed`` and ``fixed`` are aliases for ``ufixed128x18`` and ``fixed128x18``, respectively.
 
+Fixed point types are used to represent fractional numbers with a fixed precision. The number ``1.5`` is stored in a ``fixed8x2`` (8 bit number with
+two fractional digits) as ``1.50``. In the EVM, it is stored exactly the same way as the number ``150`` would be stored in an ``int8``.
+The largest and the smallest number representable in a ``fixed8x2`` are ``1.27`` and ``-1.28``, respectively.
```

Comment:
If `fixed8x2` can represent numbers between `-1.28` and `1.27` then `1.5` is out of range. The example should use a smaller one:
```suggestion
Fixed point types are used to represent fractional numbers with a fixed precision. The number ``1.23`` is stored in a ``fixed8x2`` (8 bit number with
two fractional digits) as ``1.23``. In the EVM, it is stored exactly the same way as the number ``123`` would be stored in an ``int8``.
The largest and the smallest number representable in a ``fixed8x2`` are ``1.27`` and ``-1.28``, respectively.
```

Reply:
haha good catch! :)

Reply:
I would still try to have a trailing zero

Reply:
Haha, nice!

Reply:
Good point. 1.20 or something might be better then.